packages/zowe-explorer/README.md

```diff
@@ -23,23 +23,28 @@ More information:
 
 ## Contents
 
-- [What's new in Zowe Explorer 1.12.1](#whats-new-in-zowe-explorer-1.12.1)
+- [What's new in Zowe Explorer 1.13.0](#whats-new-in-zowe-explorer-1.13.0)
 - [Prerequisites](#prerequisites)
 - [Getting Started](#getting-started)
 - [Sample Use Cases](#sample-use-cases)
 - [Credentials Security](#credentials-security)
 - [Usage Tips](#usage-tips)
 - [Extending Zowe Explorer](#extending-zowe-explorer)
 
-## What's new in Zowe Explorer 1.12.1
+## What's new in Zowe Explorer 1.13.0
 
-Bug Fixes:
+Enhancements:
 
-- Fixed the issue that prevented edited profile base paths from being saved.
-- Fixed the issue that prevented Zowe Explorer from storing empty values for optional profile fields, such as `user`, `password`, `timeout`, and `encoding`. This is done to be consistent with the way Zowe CLI stores profile information when creating and editing profiles.
-- Fixed the issue that caused repeated credential prompting if a user refused to authenticate.
-- Fixed the issue that caused removed favorite profiles to persist between IDE sessions.
-- Fixed the issue that prevented updated credential prompting from occurring when a profile was marked “invalid”.
+- Added the monorepo landing Readme that contains the high-level overview of the repository folders such as `packages` folder, instructions on how to contribute to the project and links to Medium articles providing additional useful information about Zowe Explorer and Zowe.
+- Added the previously selected `Reject Unauthorized` value to the placeholder text of the entry field while updating an existing profile.
```

Comment:
This is actually a bug fix. It loads the current RejectUnauthorized value from your profile and also bring it to the top of the selection and highlight it.